src/components/application_manager/src/resumption/resumption_data_processor_impl.cc

```diff
@@ -719,8 +719,8 @@ void ResumptionDataProcessorImpl::DeleteChoicesets(
       application->app_id(), resumption_status_, resumption_status_lock_);
 
   auto accessor = application->choice_set_map();
-  const auto& choices = accessor.GetData();
-  for (const auto& choice : choices) {
+  const auto choices = accessor.GetData();
+  for (const auto choice : choices) {
```

Comment:
```suggestion
  for (const auto& choice : choices) {
```

Reply:
fixed in e6e5767ef10ee27415d93deeac04dd30739b2f6e